compose.env

```diff
@@ -17,14 +17,14 @@ INFINITY_RERANKING_MODEL=mixedbread-ai/mxbai-rerank-xsmall-v1
 
 ## COGNITA_BACKEND VARS
 OLLAMA_URL=http://ollama-server:11434
-INFINITY_URL=http://infinity-server:7797
+INFINITY_URL=http://infinity-server:7997
 COGNITA_BACKEND_PORT=8000
 METADATA_STORE_CONFIG='{"provider":"prisma"}'
 VECTOR_DB_CONFIG='{"provider":"qdrant","url":"http://qdrant-server:6333", "config": {"grpc_port": 6334, "prefer_grpc": false}}'
 
 ## COGNITA_FRONTEND VARS
 COGNITA_FRONTEND_PORT=5001
-VITE_QA_FOUNDRY_URL=http://localhost:8000
+VITE_QA_FOUNDRY_URL=http://cognita-backend:8000
```

Comment:
Front-end for some reason is not able to connect to `cognit-backend` hence had to add localhost.

Reply:
Why? It should be able to connect, they are part of the same bridge network

I don't understand how will it work via localhost, unless frontend has requirement to talk to backend via host network.
Need to understand what breaks here

Reply:
I am able to use the UI just fine with these settings

Reply:
Intresting, we can check on this... @sayan-truefoundry and I had issues.